### ISSUE TITLE
Cannot upload profile pictures

### ISSUE THREAD

**Issue by EchedelleLR** [wintermeyer/vutuv#718]
It seems every time I fill the form to upload a profile picture in JPEG with .jpg extension, the web server returns "502 Bad Gateway" error page.

I didn't test it with other file formats, just give the information in the case it's related.

I attach a screenshot as a proof:
![Captura de pantalla de 2021-03-08 19-21-18](https://user-images.githubusercontent.com/56733813/110370382-7dff9c80-8043-11eb-8c60-907d6e85b36f.png)

**Comment by yuvrajsab** [comment]
Yes same thing here, tried with .JPG and .PNG and with different sizes too, nothing worked same error.

**Comment by diogogomes** [comment]
@EchedeyLR can you add *.png to the title? I think it's related.

 Thank you

**Comment by Danie10** [comment]
Still have the issue in Dec 2021 for .jpg. Is there some specific format we need to use, as nothing was stated during the upload?

**Comment by EchedelleLR** [comment]
I tested WebP and also doesn't work.

I have no idea which format is allowed of if the issue affects the infrastructure itself preventing the upload of any profile picture.

**Comment by ghost** [comment]
Yes I still can't post my profile pictures today (Dec 21, 2021)

**Comment by hsebert** [comment]
Same error here (Nov 29, 2022)

*Tap*, *tap* – Is this thing on??

**Comment by wintermeyer** [comment]
> Tap, tap – Is this thing on??

No sugarcoating from my side: The bug is still not fixed. I know. 🤦‍♂️

**Comment by beedell-roke** [comment]
Similarly to what [`issues/750`](https://github.com/wintermeyer/vutuv/issues/750#issue-2622073577) describes:

1. I attempted to upload https://github.com/user-attachments/assets/d61cada1-d44c-41a3-b2ec-2370464742d4 at https://www.vutuv.de/users/roke.beedell/edit#:~:text=Beedell%20/%20Edit-,Avater,-Gender.

3. Was redirected to https://www.vutuv.de/users/roke.beedell#:~:text=502%20Bad%20Gateway,nginx/1.22.1, wherein `view-source:https://www.vutuv.de/users/roke.beedell#:~:text=%3Chtml%3E,%3C/html%3E` states:

	<blockquote>

	```HTML
	<html>
	<head><title>502 Bad Gateway</title></head>
	<body>
	<center><h1>502 Bad Gateway</h1></center>
	<hr><center>nginx/1.22.1</center>
	</body>
	</html>
	
	```

	</blockquote>

To demonstrate this:

https://github.com/user-attachments/assets/f8b7a2d7-2fd8-44e4-87a0-5dd1b76b643e
